keyboards/xd60/keymaps/melka/config.h

```diff
@@ -0,0 +1,14 @@
+#include "passwd.h"
```

Comment:
This may be a better way to call this. 

```suggestion
#pragma once

#if __GNUC__ > 5
#    if __has_include("passwd.h")
#        include "passwd.h"
#    endif  // if file exists
#endif      // __GNUC__
```

Specifically, GCC 6 and up support `__has_include`, only, so only call it if it's supported.  And then `__has_include` will actually check to see if the file exists. 

I use this for a couple of things in my userspace.